libvirt/tests/src/passthrough/pci/libvirt_pci_passthrough_hotplug.py

```
@@ -85,19 +100,38 @@ def run(test, params, env):
+
     devices.append(controller)
     vmxml.set_devices(devices)
     vmxml.sync()
     if not vm.is_alive():
         vm.start()
         session = vm.wait_for_login()
-    if not utils_package.package_install(["ppc64-diag",
-                                          "librtas", "powerpc-utils"],
-                                         session, 360):
+    detected_distro = distro.detect()
+    distro_name = detected_distro.name.lower()
```

Ruff (0.13.3)

Local variable `distro_name` is assigned to but never used

Remove assignment to unused variable `distro_name`

(F841)